components/logins/src/db.rs

```diff
@@ -24,22 +45,22 @@ use url::{Host, Url};
 
 #[derive(Serialize, Deserialize, PartialEq, Debug, Clone, Default)]
 pub struct MigrationPhaseMetrics {
-    num_processed: u64,
-    num_succeeded: u64,
-    num_failed: u64,
-    total_duration: u128,
-    errors: Vec<String>,
+    pub num_processed: u64,
```

Comment:
It would be good if these could be `pub(crate)` instead - that's what the uniffi branch has ended up with, so we might as well avoid future conflicts early.

...

Actually, that branch no longer needs them to be public in any way! I asked Sammy to revert that change in his branch.

I also see that sync-pass uses them. TBH, I'd be inclined to have sync-pass just `{:?}` the entire struct - it's really just for us. But I'll leave that up to you!

Reply:
Sounds good.  I'll update sync-pass to use `{:?}` and add a note to update the field visibility once the uniffi branch gets merged in.